pkg/kubelet/rkt/rkt.go

```diff
@@ -81,7 +81,7 @@ const (
 	k8sRktCreationTimeAnno           = "rkt.kubernetes.io/created"
 	k8sRktContainerHashAnno          = "rkt.kubernetes.io/containerhash"
 	k8sRktRestartCountAnno           = "rkt.kubernetes.io/restartcount"
-	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/terminationMessagePath"
+	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/terminationmessagepath"
```

Comment:
I agree that `terminationmessagepath` is getting a bit long. But it seems to be the standard here.

I would be reluctant to change it to `termination-message-path` without changing the rest of them, `restartcount` to `restart-count` for example.